animations/lib/main.dart

```diff
@@ -147,29 +149,38 @@ final miscDemos = [
       builder: (context) => const CurvedAnimationDemo()),
 ];
 
-final basicDemoRoutes =
-    Map.fromEntries(basicDemos.map((d) => MapEntry(d.route, d.builder)));
-
-final miscDemoRoutes =
-    Map.fromEntries(miscDemos.map((d) => MapEntry(d.route, d.builder)));
-
-final allRoutes = <String, WidgetBuilder>{
-  ...basicDemoRoutes,
-  ...miscDemoRoutes,
-};
+final router = GoRouter(
+  routes: [
+    GoRoute(
+      path: '/',
+      builder: (context, state) => const HomePage(),
+      routes: [
+        for (final demo in basicDemos)
+          GoRoute(
+            path: demo.route.substring(1),
```

Comment:
Need to remove the starting `/` from the original routes.

If you think this is a bit too hacky, I can go ahead and modify all the route names inside their classes.

Reply:
I'd vote to update the `route` values instead.